FlowCal/mef.py

```diff
@@ -71,6 +76,19 @@ def clustering_gmm(data,
     events. For more information, consult ``scikit-learn``'s documentation.
 
     """
+    # Check that ``scale`` is supported
+    if scale!='linear' and scale!='log':
+        raise ValueError("scale {} not supported".format(scale))
+
+    # Rescale if necessary
+    if scale=='log':
+        # Copy events
+        data = data.copy()
+        # Log scale requires saturating negative and zero-valued events
+        data[data < 1] = 1.
+        # Rescale
+        data = np.log10(data)*256
```

Comment:
Apparently changing the `min_covar` parameter from `1e-2` to `5e-5` (rounded from `1e-2/256 ~= 3.9e-5`) was enough to compensate for removing the `*256`. 352b8f4d47952e4e7c861e69c1e2d9e1b283aca5 implements this.